apps/judicial-system/web/src/routes/Court/InvestigationRequest/CourtRecord/CourtRecordForm.tsx

```diff
@@ -233,8 +233,7 @@ const CourtRecordForm: React.FC<Props> = (props) => {
             workingCase={workingCase}
           />
         </Box>
-        {workingCase.sessionArrangements ===
-          SessionArrangements.ALL_PRESENT && (
+        {workingCase.accusedBookings && (
```

Comment:
Isn't this a bit dangerous? If the user deletes all text from accusedBookings, then the input field will disappear. I think you must use `workingCase.sessionArrangements` in this condition.